04/app.js

```diff
@@ -6,4 +6,26 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const currentDate = new Date();
+const currentDay = currentDate.getDate();
+const currentMonth = currentDate.getMonth() + 1;
+
+
+function checkBirthday(user){
+
+    const userBirthday = user.born;
+    const birthdayDay = parseInt(userBirthday.day);
+    const birthdayMonth = parseInt(userBirthday.month);
+
+    if (birthdayDay == currentDay && birthdayMonth == currentMonth) {
+        return 'Wszystkiego najlepszego z okazji dzisiejszych urodzin!';
+    }
+    else {
+        return 'Dzis nie masz urodzin'
```

Comment:
Lepiej zwrócić true/fasle, a potem zwracać komunikat, bo może być on np. różny - co innego napiszesz babci, co innego dziecku, a jeszcze inaczej partnerce ;)

PS. W szczególności, że nazwa funkcji wskazuje sprawdzenie tak/nie - zamiast konkretnego komunikatu. Oczywiście się czepiam, ale w przyszłości może to być pomocne ;P